components/automate-cli/cmd/chef-automate/certRotate.go

```diff
@@ -169,12 +181,25 @@ func certRotatePG(publicCert, privateCert, rootCA string, infra *AutomteHAInfraD
 	remoteService := "postgresql"
 
 	// Creating and patching the required configurations.
-	config := fmt.Sprintf(POSTGRES_CONFIG, privateCert, publicCert, rootCA)
+	var config string
+	if nodeFlag.node != "" {
+		if rootCA != "" {
+			writer.Warn("root-ca flag will be ignored when node flag is provided")
```

Comment:
this should be not only for PG but remaining as well. so place it in the common code i.e. `certRotate()` method.

Reply:
updated.